dotcom-rendering/src/web/lib/decidePalette.ts

```diff
@@ -57,6 +70,18 @@ const textHeadline = (format: ArticleFormat): string => {
 				format.design !== ArticleDesign.Interview
 			)
 				return specialReport[100];
+
+			if (format.theme === ArticleSpecial.SpecialReportAlt) {
+				if (format.design === ArticleDesign.Interview)
+					return palette.specialReportAlt[800];
```

Comment:
This is the new way of referring to `source` `palette`: https://github.com/guardian/source/blob/main/packages/%40guardian/source-foundations/src/index.ts#L22